styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: white;
+}
+
+.ejemplo{
+    display: flex;
+    justify-content: center;
+	align-items:center;
+	flex-direction: row;
+}
+
+.grad {
+	background: linear-gradient(to top, transparent, orange, transparent);
+	margin: 100px 40px 100px 40px;
```

Comment:
this could be reduced to:
```
margin: 100px 40px;
```
the first value goes for the top and bottom and the second value goes to the right and left values